history.md

```diff
@@ -1,5 +1,25 @@
 # History
 
+# 1.9.2
```

Comment:
lot of new features so this should be 1.10.0

Reply:
Done